integration/tracing/tracing-opentelemetry/src/main/java/org/apache/cxf/tracing/opentelemetry/OpenTelemetryTracerUtils.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.cxf.tracing.opentelemetry;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Tracer;
+import org.apache.cxf.version.Version;
+
+public final class OpenTelemetryTracerUtils {
```

Comment:
I would inclined to drop this class and use constant for default name but also provide the way to specify the tracer name (fe `BraveFeature` follows this approach). Please let me know what do you think, I am happy to take that as post merge action, thank you.

Reply:
I removed the utility class, and added new constructors with an instrumentationName argument. It defaults to a constant "org.apache.cxf". I remove all notions of instrumentation versions (looking in opentelemetry-java-instrumentation, I see that it is not always set). Custom names are tested in the systests.